blueflood-core/src/main/java/com/rackspacecloud/blueflood/service/UpdateStamp.java

```diff
@@ -35,21 +35,32 @@
      * database.
      */
     private boolean dirty;
-    
+
+    private long lastRollupTimestamp;
+
     public UpdateStamp(long timestamp, State state, boolean dirty) {
         setTimestamp(timestamp);
         setState(state);
         setDirty(dirty);
     }
```

Comment:
It is still being used. This is the case of a finding a slot for the first time during ingest which is not present in metrics_state table yet. This will probably happen only during dev.

https://github.com/rackerlabs/blueflood/blob/f24a8ba3990dd38e26e29da1d3ec234285c478f1/blueflood-core/src/main/java/com/rackspacecloud/blueflood/service/ShardStateManager.java#L270